### PR TITLE
Homepage css

### PR DESCRIPTION
**Pull request by PrettyPearl2302**
## Description

initial css for homepage

## Milestones

homepage display

## Test Plan
pull3

### CODE REVIEW

**Comment by jeffwdeng** [comment]
Would benefit from more description on what we're trying to accomplish with the changes to home page in the description/title

File: timify-ui/public/stores1.png
Comment (ssorenson130) [inline timify-ui/public/stores1.png:1]: huh? was this accidentally copied over from somewhere?

File: timify-ui/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import './Footer.css';
+
+const Footer = () => {
+  return (
+    <>
+      <footer className="footer">
+      <div className='footer-wrapper'></div>
+        <div className="footer-section">
+          <div>
+              <h4>Company</h4>
```
Comment (ssorenson130) [inline timify-ui/src/components/Footer/Footer.jsx:10]: same here. why do we have things like a careers page, blog, etc?

**Review by ssorenson130 — CHANGES_REQUESTED** [review]

File: timify-ui/src/components/Footer/Footer.jsx (hunk 1 above)
Comment (PrettyPearl2302) [reply timify-ui/src/components/Footer/Footer.jsx:10]: yeah it's the footer from my student store. i forgot to mention in my resources. i was kind of just doing placeholder stuff but i can write the actual things now

File: timify-ui/src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+        <div className="footer-section">
+          <div>
+              <h4>Socials</h4>
+              <div>
+                  <p>Facebook</p>
+                  <p>Twitter</p>
+                  <p>Linkedin</p>
+                  <p>Instagram</p>
+                  <p>Youtube</p>
```
Comment (jeffwdeng) [inline timify-ui/src/components/Footer/Footer.jsx:38]: i think the question applies here as well. what are the intention of the socials?

Comment (PrettyPearl2302) [reply timify-ui/src/components/Footer/Footer.jsx:38]: same reason

File: timify-ui/src/components/Footer/Footer.jsx (hunk 1 above)
Comment (ssorenson130) [reply timify-ui/src/components/Footer/Footer.jsx:10]: ah ok. next time include a little more info so we understand the source.

**Review by ssorenson130 — APPROVED** [review]
we can follow up with fixing all the footer content

**Pull request merged**